### PR TITLE
Add EIP: ETH (Native Asset) Address Convention

### PR DESCRIPTION
**Pull request by Joeysantoro**
Discussions to: https://ethereum-magicians.org/t/eip-7808-eth-native-asset-address-convention/15989

### CODE REVIEW

File: EIPS/eip-7808.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+---
+eip: 7808
```
Comment (abcoathup) [inline EIPS/eip-7808.md:2]:
```suggestion
eip: 7528
```
Assigning next sequential EIP/ERC number.
EIP/ERC numbering changed to sequential from 7500 and is no longer the PR number.

Please also update the filename and the Eth Magicians discussion.

File: EIPS/eip-7808.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+title: ETH (Native Asset) Address Convention
+description: An address placeholder for ETH when used in the same context as an EIP-20 token.
+author: Joey Santoro (@joeysantoro)
+discussions-to: https://ethereum-magicians.org/t/eip-7808-eth-native-asset-address-convention/15989
+status: Draft
+type: Standards Track
+category: ERC
+created: 2023-10-03
+requires: [EIP-20](https://eips.ethereum.org/EIPS/eip-20), [EIP-4626](https://eips.ethereum.org/EIPS/eip-4626)
```
Comment (abcoathup) [inline EIPS/eip-7808.md:11]:
```suggestion
requires: 20, 4626
```

File: EIPS/eip-7808.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+## Motivation
+
+ETH, being a fungible unit of value, often behaves similarly to [EIP-20](https://eips.ethereum.org/EIPS/eip-20) tokens. Protocols tend to implement a standard interface for EIP-20 tokens, and benefit from having the ETH implementation to closely mirror the [EIP-20](https://eips.ethereum.org/EIPS/eip-20) implementations.
+
+In many cases, protocols opt to use Wrapped ETH ([WETH](https://etherscan.io/token/0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2)) for [EIP-20](https://eips.ethereum.org/EIPS/eip-20) compliance. In other cases, protocols will use native ETH due to gas considerations, or the requirement of using native ETH such as in the case of a Liquid Staking Token (LST).
+
+One intended use case for the standard is [EIP-4626](https://eips.ethereum.org/EIPS/eip-4626) compliant LSTs which use ETH as the `asset`. This extends the benefits and tooling of [EIP-4626](https://eips.ethereum.org/EIPS/eip-4626) to LSTs and integrating protocols.
+
+This standard allows protocols and off-chain data infrastructure to  coordinate around a shared understanding that any time 0xe is used as an address in an [EIP-20](https://eips.ethereum.org/EIPS/eip-20) context, it means ETH.
```
Comment (abcoathup) [inline EIPS/eip-7808.md:28]:
```suggestion
This standard allows protocols and off-chain data infrastructure to coordinate around a shared understanding that any time 0xe is used as an address in an [EIP-20](https://eips.ethereum.org/EIPS/eip-20) context, it means ETH.
```

**Comment by Joeysantoro** [comment]
@axic , @Pandapip1 , @SamWilsn , @xinbenlv this should be ready to move to Review!

**Review by lightclient — APPROVED** [review]

File: EIPS/eip-7528.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
+---
+eip: 7528
+title: ETH (Native Asset) Address Convention
+description: An address placeholder for ETH when used in the same context as an ERC-20 token.
+author: Joey Santoro (@joeysantoro)
+discussions-to: https://ethereum-magicians.org/t/eip-7808-eth-native-asset-address-convention/15989
+status: Review
```
Comment (lightclient) [inline EIPS/eip-7528.md:7]:
```suggestion
status: Draft
```

**Review by lightclient — CHANGES_REQUESTED** [review]

**Review by lightclient — APPROVED** [review]

**Pull request merged**